src/event.rs

```diff
@@ -273,6 +277,9 @@ where
                     let delta = mouse::last_delta(ctx);
                     state.mouse_motion_event(ctx, position.x, position.y, delta.x, delta.y);
                 }
+                WindowEvent::Touch(touch) => {
+                    state.touch_event(ctx, touch.phase, touch.location.x, touch.location.y);
+                }
```

Comment:
You were a bit unlucky with your timing though, as we just merged a PR extending error handling to all methods of `EventHandler`.

To make the new touch event fit that scheme you'd have to adapt this code like this:
```suggestion
                WindowEvent::Touch(touch) => {
                    let res = state.touch_event(ctx, touch.phase, touch.location.x, touch.location.y);
                    if catch_error(ctx, res, state, control_flow, ErrorOrigin::TouchEvent) {
                        return;
                    };
                }
```

Of course, you'd also have to add `TouchEvent` to the `ErrorOrigin` enum.